instrumentation/active_job/test/test_helper.rb

```diff
@@ -42,12 +42,13 @@ def perform
 class PositionalOnlyArgsJob < ::ActiveJob::Base
   def perform(arg1, arg2 = 'default'); end
 end
+
 class KeywordOnlyArgsJob < ::ActiveJob::Base
-  def perform(keyword1: 'default', keyword2:); end
+  def perform(keyword2:, keyword1: 'default'); end
 end
 
 class MixedArgsJob < ::ActiveJob::Base
-  def perform(arg1, arg2, keyword1: 'default', keyword2:); end
+  def perform(arg1, arg2, keyword2:, keyword1: 'default'); end
```

Comment:
Why?

Reply:
I'll posit that it was to appease [the Style/KeywordParametersOrder cop](https://docs.rubocop.org/rubocop/cops_style.html#stylekeywordparametersorder) where required keyword params come first and optional all appear at the end of the params list.

Reply:
yep